pyproject.toml

```diff
@@ -1,18 +1,59 @@
 [build-system]
-requires = [
-    "scikit-build>=0.12",
-    "setuptools>=42",
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "cmake"
+version = "0.0.1"
```

Comment:
I guess dynamic versions are for the future ?

Reply:
Pretty low priority for now, as it adds significant extra complication for very little benefit for now. Eventually will probably add support for reading from a regex & setuptools_scm.